docs/oauth2/server.rst

```diff
@@ -246,6 +246,17 @@ the token.
 
         expires_at = django.db.models.DateTimeField()
 
+**PKCE Challenge (optional)**
+
+    If you want to support PKCE, you have to associate a `code_challenge`
+    and a `code_challenge_method` to the actual Authorization Code.
+
+    .. code-block:: python
+
+        challenge = django.db.models.CharField(max_length=100)
```

Comment:
When `code_challenge_method=plain`, `code_challenge` = `code_verifier` where `code_verifier` has a minimum length of 43 and a maximum length of 128 - https://tools.ietf.org/html/rfc7636#section-4.1